### PR TITLE
update README.md

### PR DESCRIPTION
**Pull request by shanluo911**
[AB#358291](https://dev.azure.com/ceapex/d3d54af3-265a-4f18-95f6-9a46397ca583/_workitems/edit/358291)

### CODE REVIEW

**Review by 928PJY — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -12,22 +12,60 @@ This extension enables you to run build validation on a Docs conceptual or Learn
...
 ![OAuth](https://github.com/docascode/vscode-docs-build/blob/master/resources/vscode-docs-build.gif?raw=true)
 
 > **Note:** The first time you validate a repo, all the Docs build dependencies will be fetched and cached locally. Subsequent validation runs will be faster.
 
+### Real-time validation
+
+1. Open a Docs conceptual or Learn repo in VS Code with the extension installed.
+2. For the first time you use the extension, you will be suggested to choose your user type between **Microsoft employee** and **Publish contributor** before you can use real-time validation.
+3. For **Microsoft employee**, you are required to sign in before using real-time validation. You can click **Docs Validation** on the status bar then click **Sign in** from the drop-down menu to sign in.
+4. The real-time validation is disabled by default, you can enable it in the extension settings. You will be suggested to reload the extension after you enable real-time validation.
```
Comment (herohua) [inline README.md]: `You will be asked to reload the extension after you enable real-time validation in the settings.`

**Review by herohua — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -12,22 +12,60 @@ This extension enables you to run build validation on a Docs conceptual or Learn
...
-1. Trigger a build by clicking **Yes** when prompted after sign in, by clicking **Docs Validation** on the status bar then clicking **Validate** from the drop-down menu, or by right-clicking any file in the repo and selecting **Validate this repository**.
+1. For the first time you use the extension, you will be suggested to choose your user type between **Microsoft employee** and **Public contributor** before you can use full-repository validation.
+1. For **Microsoft employee**, you are required to sign in before using full-repository validation. You can click **Docs Validation** on the status bar then click **Sign in** from the drop-down menu to sign in.
+1. After signing in, you can trigger a validation by clicking **Yes** when prompted. You can also click **Docs Validation** on the status bar then click **Validate** from the drop-down menu, or by right-clicking any file in the repo and selecting **Validate this repository**.
```
Comment (928PJY) [inline README.md]: do we have a `Yes` button?

File: README.md [hunk 3]
```diff
@@ -12,22 +12,60 @@ This extension enables you to run build validation on a Docs conceptual or Learn
 
 ## How to use the Docs Validation extension
 
+### Full-repository validation
+
 1. Open a Docs conceptual or Learn repo in VS Code with the extension installed.
-1. If you're a Microsoft employee, optionally sign in in by clicking **Docs Validation** from the VS Code status bar. You can still use the extension without signing in, but sign in is recommended to use the most recent build information for private repos.
-1. Trigger a build by clicking **Yes** when prompted after sign in, by clicking **Docs Validation** on the status bar then clicking **Validate** from the drop-down menu, or by right-clicking any file in the repo and selecting **Validate this repository**.
+1. For the first time you use the extension, you will be suggested to choose your user type between **Microsoft employee** and **Public contributor** before you can use full-repository validation.
```
Comment (928PJY) [inline README.md]: `you will be asked to choose..`

File: README.md [hunk 4]
```diff
@@ -12,22 +12,60 @@ This extension enables you to run build validation on a Docs conceptual or Learn
...
 
 ![OAuth](https://github.com/docascode/vscode-docs-build/blob/master/resources/vscode-docs-build.gif?raw=true)
 
 > **Note:** The first time you validate a repo, all the Docs build dependencies will be fetched and cached locally. Subsequent validation runs will be faster.
 
+### Real-time validation
+
+1. Open a Docs conceptual or Learn repo in VS Code with the extension installed.
+2. For the first time you use the extension, you will be suggested to choose your user type between **Microsoft employee** and **Publish contributor** before you can use real-time validation.
+3. For **Microsoft employee**, you are required to sign in before using real-time validation. You can click **Docs Validation** on the status bar then click **Sign in** from the drop-down menu to sign in.
```
Comment (928PJY) [inline README.md]: This may confuse the user that they may need to `sign-in` manually.

File: README.md (hunk 1 above)
Comment (928PJY) [inline README.md]: let's also add an instruction about how to enable the flag

File: README.md (hunk 2 above)
Comment (shanluo911) [reply README.md]: change to **Validate**

**Review by 928PJY — COMMENTED** [review]
🏎

**Review by 928PJY — CHANGES_REQUESTED** [review]
🏎

**Review by 928PJY — APPROVED** [review]

**Pull request merged**